test/integration/integration-test.ts

```diff
@@ -348,16 +350,6 @@ describe('integration tests', function () {
     })
   })
 
-  it('ticket', function () {
-    return this.api.getLedgerVersion().then((ledgerVersion) => {
-      return this.api
-        .prepareTicketCreate(address, 1, instructions)
-        .then((prepared) =>
-          testTransaction(this, 'ticket', ledgerVersion, prepared)
-        )
-    })
-  })
-
```

Comment:
Why is this test removed?

Reply:
I can enable the amendment manually on my standalone node, and then this test will work.